infra/network.yaml

```diff
@@ -119,6 +119,21 @@ Resources:
     Properties:
       SubnetId: !Ref 'privsubnet2'
       RouteTableId: !Ref 'privroutetable2'
+  s3endpoint:
+    Type: AWS::EC2::VPCEndpoint
+    Properties:
+      PolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Effect: Allow
+            Principal: '*'
+            Action: '*'
+            Resource: '*'
+      RouteTableIds:
+        - !Ref 'privroutetable1'
+        - !Ref 'privroutetable2'
+      ServiceName: !Sub 'com.amazonaws.${AWS::Region}.s3'
```

Comment:
*question*: Do we use AWS::Region elsewhere, or did I hardcode us-west-2 by accident?